packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1332,6 +1332,22 @@ export const voicecraft = (model: ModelData): string[] => [
 model = VoiceCraft.from_pretrained("${model.id}")`,
 ];
 
+export const vui = (model: ModelData): string[] => [
+	`import torchaudio
+
+from vui.inference import render
+from vui.model import Vui,
+
+model = Vui.from_pretrained().cuda()
```

Comment:
typically here if repo is split in several models, we could have a better `.from_pretrained` snippet. Requires some slight changes in the GH repo as well but worth it IMO